lady_luck.js

```diff
@@ -230,8 +206,7 @@ function generateTableContent(initTable) {
 
 function nwodInitForceToText(msg, val, name) {
 	if (!/^\d+$/.test(val)) {
-		msg.reply("Can only force the init table if given a number");
-		return;
+		return msg.reply("Can only force the init table if given a number");
```

Comment:
What are you hoping for this return value to be used for? Running it and returning is more correct.